airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -544,6 +544,21 @@ def await_xcom_sidecar_container_start(self, pod: V1Pod) -> None:
 
     def extract_xcom(self, pod: V1Pod) -> str:
         """Retrieves XCom value and kills xcom sidecar container."""
+        try:
+            result = self.extract_xcom_json(pod)
+            return result
+        except Exception as ex:
+            raise ex
```

Comment:
Following https://docs.python.org/3/tutorial/errors.html#defining-clean-up-actions this is not needed - or is there a reason that you re-raise the same?
```suggestion
```

Reply:
Thx! removed except block